runtime/router.go

```diff
@@ -134,77 +58,60 @@ func NewRouterEndpoint(
 	handlerName string,
 	handler HandlerFn,
 ) *RouterEndpoint {
-	endpointTags := map[string]string{
+	scope := gateway.AllHostScope.Tagged(map[string]string{
 		"endpoint": endpointName,
 		"handler":  handlerName,
-	}
-	endpointScope := gateway.AllHostScope.Tagged(endpointTags)
-	requestRecvd := endpointScope.Counter("inbound.calls.recvd")
-	requestLatency := endpointScope.Timer("inbound.calls.latency")
-	statusCodes := make(map[int]tally.Counter, len(knownStatusCodes))
-
-	for _, statusCode := range knownStatusCodes {
-		metricName := "inbound.calls.status." + strconv.Itoa(statusCode)
-		statusCodes[statusCode] = endpointScope.Counter(metricName)
-	}
-
+	})
 	return &RouterEndpoint{
-		EndpointName: endpointName,
```

Comment:
is the endpoint name already added to the logger?

Reply:
That will be done in a separate PR for logger changes.

Reply:
Then wondering if you want to remove endpointName from the struct.

Reply:
This particular EndpointName field was not used by the logger.

Reply:
The question is specifically to ensure that we are able to tag the logger with endpoint context and if it would be used in future, so wondering why do we need to remove it now.

Reply:
The plan is to pass in `map[string]string` to `NewRouterEndpoint()` and use that to create tagged scope and add fields to logger.